webpack/components/Nav.jsx

```diff
@@ -13,7 +13,7 @@ export default class Nav extends React.Component {
     });
 
     return (
-      <nav className="navbar navbar-fixed-top">
+      <nav className="navbar navbar-default">
```

Comment:
This will be addressed in the pull request for #10. Can it be removed from this pull request pretty please?